src/lib/transaction_logic/mina_transaction_logic.ml

```diff
@@ -2398,7 +2400,7 @@ module For_tests = struct
                   ; sequence_events = []
                   ; call_data = Snark_params.Tick.Field.zero
                   ; call_depth = 0
-                  ; protocol_state = Snapp_predicate.Protocol_state.accept
+                  ; protocol_state = Zkapp_precondition.Protocol_state.accept
                   ; use_full_commitment = ()
                   }
               ; predicate = actual_nonce
```

Comment:
I wonder if we should rename the `predicate` field to `account_precondition` for clarity/consistency? (in a different PR)

Reply:
Definitely, yes